src/lib/runtime/Web3FunctionRunner.ts

```diff
@@ -159,26 +161,34 @@ export class Web3FunctionRunner {
     const start = performance.now();
     const throttled: Web3FunctionThrottled = {};
     let result: Web3FunctionResult | undefined = undefined;
+    let callbacks: Web3FunctionCallbackStatus | undefined = undefined;
     let storage: Web3FunctionStorageWithSize | undefined = undefined;
     let error: Error | undefined = undefined;
+
     const { script, context, options, version, multiChainProviderConfig } =
       payload;
+
     try {
-      const data = await this._runInSandbox(
+      const { data } = await this._runInSandbox(
         script,
         version,
         context,
         options,
         multiChainProviderConfig
       );
-      this._validateResult(version, data.result);
+
+      if (context.operation === "onRun") {
+        this._validateResult(version, data.result as Web3FunctionResult);
+      }
 
       result = data.result;
+      callbacks = data.callbacks;
       storage = {
         ...data.storage,
         size: Buffer.byteLength(JSON.stringify(data.storage), "utf-8") / 1024,
       };
     } catch (err) {
+      console.log("ERROR: ", err);
```

Comment:
Shall we remove this log, it displays a log on timeout errors as well on cloud runner, wdyt? @brandonchuah 
<img width="1162" alt="Ekran Resmi 2023-11-30 15 05 57" src="https://github.com/gelatodigital/web3-functions-sdk/assets/3192210/2a19df59-0264-47e7-a478-54b37ed49222">